std/mmfile.d

```diff
@@ -219,7 +219,7 @@ class MmFile
 
             if (filename != null)
             {
-                hFile = CreateFileW(filename.tempCStringW(),
+                hFile = CreateFileW(cast(LPCWSTR) filename.tempCStringW(),
```

Comment:
Change `tempCStringW` to return `LPWSTR` instead of `wchar*`?

Reply:
Perhaps, but I don't know what other things tempCStringW might be used for. It's defined in internal/cstring.d, and may be worth changing.

Reply:
I think it would be a very safe bet to assume that `tempCStringW` was created to help interfacing with Unicode Windows APIs.